src/main/scala/cognite/spark/v1/PushdownUtilities.scala

```diff
@@ -149,10 +149,10 @@ object PushdownUtilities {
 
   def getTimestampLimit(filter: Filter, colName: String): Seq[Limit] =
     filter match {
-      case LessThan(colName, value) => Seq(timeStampStringToMax(value, -1))
-      case LessThanOrEqual(colName, value) => Seq(timeStampStringToMax(value, 0))
-      case GreaterThan(colName, value) => Seq(timeStampStringToMin(value, 1))
-      case GreaterThanOrEqual(colName, value) => Seq(timeStampStringToMin(value, 0))
+      case LessThan(`colName`, value) => Seq(timeStampStringToMax(value, -1))
```

Comment:
why "`" ?

Reply:
Oh, that is a Scala trap: https://stackoverflow.com/a/7078077

Before, it declared a new variable instead of checking that it's equal to colName...